### ISSUE TITLE
Task :@react-native-community_checkbox:compileDebugJavaWithJavac FAILED

### ISSUE THREAD

**Issue by BravenxX** [react-native-community/react-native-checkbox#27]
Suddenly I have a problem with this library , I don't know what happened but now it gets me this error when I do: react-native run-android

> Task :@react-native-community_checkbox:compileDebugJavaWithJavac FAILED

**FULL ERROR**
```
> Configure project :@react-native-community_checkbox
WARNING: Configuration 'compile' is obsolete and has been replaced with 'implementation' and 'api'.
It will be removed soon. For more information see: http://d.android.com/r/tools/update-dependency-configurations.html
WARNING: The specified Android SDK Build Tools version (23.0.1) is ignored, as it is below the minimum supported version (28.0.3) for Android Gradle Plugin 3.5.2.
Android SDK Build Tools 28.0.3 will be used.
To suppress this warning, remove "buildToolsVersion '23.0.1'" from your build.gradle file, as each version of the Android Gradle Plugin now has a default version of the build tools.

> Task :@react-native-community_checkbox:compileDebugJavaWithJavac FAILED
9 actionable tasks: 1 executed, 8 up-to-date

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':@react-native-community_checkbox:compileDebugJavaWithJavac'.
> Could not find tools.jar. Please check that C:\Program Files\Java\jre1.8.0_231 contains a valid JDK installation.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 4s

error Failed to install the app. Make sure you have the Android development environment set up: https://facebook.github.io/react-native/docs/getting-started.html#android-development-environment. Run CLI with --verbose flag for more details.
Error: Command failed: gradlew.bat app:installDebug -PreactNativeDevServerPort=8081

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':@react-native-community_checkbox:compileDebugJavaWithJavac'.
> Could not find tools.jar. Please check that C:\Program Files\Java\jre1.8.0_231 contains a valid JDK installation.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 4s
```

**My specs:** 

    "@react-native-community/checkbox": "^0.2.1",
    "react": "^16.10.2",
    "react-native": "^0.61.2",

**Comment by rahul2992** [comment]
Any update on this? I am getting the same error. Thank you

**Comment by BravenxX** [comment]
> Any update on this? I am getting the same error. Thank you

I'm still with the same problem...

**Comment by Naturalclar** [comment]
@BravenxX 
Have you tried reinstalling the jdk?
There seems to be similar issues https://github.com/facebook/react-native/issues/16844 around and that seems to be the way people solve this particular error.

**Comment by BrianHlcn** [comment]
have you tried with "npx jetify" command? it worked for me

**Comment by muhamadabduh** [comment]
> have you tried with "npx jetify" command? it worked for me

it worked for me too, but I also edited android/app/build.gradle, my RN version 0.61.5
- implementation "com.facebook.react:react-native:+"   
+ implementation "com.facebook.react:react-native:0.61.5"
